### PR TITLE
[CoW Amm Deployer] Add stop AMM feature

### PR DESCRIPTION
**Pull request by yvesfracari**

### CODE REVIEW

File: apps/cow-amm-deployer/src/app/amm/HomePageWrapper.tsx [hunk 1]
```diff
@@ -114,7 +119,14 @@ export function HomePageWrapper() {
             <Button
               className="flex items-center gap-1 py-3 px-6 "
               color="tomato"
-              disabled
+              onClick={async () => {
+                await sendTransactions([
+                  {
+                    type: TRANSACTION_TYPES.STOP_COW_AMM,
```
Comment (ribeirojose) [inline apps/cow-amm-deployer/src/app/amm/HomePageWrapper.tsx:125]: 🙆‍♂️ cool! I'm assuming all this had already been implemented then?

File: apps/cow-amm-deployer/src/lib/transactionFactory.ts [hunk 2]
```diff
@@ -140,11 +145,25 @@ class CowAmmCreateTx implements ITransaction<creteCowAmmArgs> {
     };
   }
 }
+
+class CowAmmStopTx implements ITransaction<stopCowAmmArgs> {
```
Comment (ribeirojose) [inline apps/cow-amm-deployer/src/lib/transactionFactory.ts:149]: any comments on why use `Stop` when the fn name is `Close`? Do you think that's more friendly/meaningful to users?

**Review by ribeirojose — APPROVED** [review]
LGTM! A few comments/questions:

Would highly encourage us to add a note/confirmation before calling AMM stop/close ensuring users their funds will still stay in the safe etc. Also curious: can they reenable it in the future or would they have to create another AMM? Perhaps we could also allow for that. How would the UI show for a user whose AMM has been stopped?

File: apps/cow-amm-deployer/src/lib/transactionFactory.ts (hunk 2 above)
Comment (yvesfracari) [reply apps/cow-amm-deployer/src/lib/transactionFactory.ts:149]: On the UI I thought that the `Stop` would be more friendly, but I don't have a strong opinion here actually

**Comment by yvesfracari** [comment]
For now, that is more like a PoC, it will be the same page when there is no AMM running. In the future, I think that we could add a list of all AMMs that the user created and he can enable them (if there isn't any other one activated)

**Pull request merged**